### PR TITLE
Fix the names for adyen configuration

### PR DESCRIPTION
**Pull request by korycins**
Fix the names for Adyen configuration

### CODE REVIEW

File: saleor/payment/gateways/adyen/plugin.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ class AdyenGatewayPlugin(BasePlugin):
             ),
             "label": "Origin Url",
         },
-        "Live": {
+        "live": {
             "type": ConfigurationTypeField.STRING,
             "help_text": (
                 "Leave it blank when you want to use test env. To communicate with the"
```
Comment (patrys) [inline saleor/payment/gateways/adyen/plugin.py:111]: What should I put here for a live environment?

File: saleor/payment/gateways/adyen/plugin.py [hunk 2]
```diff
@@ -105,7 +105,7 @@ class AdyenGatewayPlugin(BasePlugin):
             ),
             "label": "Origin Url",
```
Comment (patrys) [inline saleor/payment/gateways/adyen/plugin.py:106]:
```suggestion
            "label": "Origin URL",
```

File: saleor/payment/gateways/adyen/plugin.py [hunk 3]
```diff
@@ -94,7 +94,7 @@ class AdyenGatewayPlugin(BasePlugin):
             ),
             "label": "Client Key",
         },
-        "Origin Url": {
+        "origin-url": {
             "type": ConfigurationTypeField.STRING,
             "help_text": (
                 "The origin URL of the page where you are rendering the Drop-in. This "
```
Comment (patrys) [inline saleor/payment/gateways/adyen/plugin.py:100]: What's a Drop-in? Should this link to Adyen docs?

File: saleor/payment/gateways/adyen/plugin.py (hunk 1 above)
Comment (korycins) [reply saleor/payment/gateways/adyen/plugin.py:111]: I've added here the link to docs - `https://docs.adyen.com/development-resources/live-endpoints`. line 116. Is it not enough?

Comment (patrys) [reply saleor/payment/gateways/adyen/plugin.py:111]: 👍

File: saleor/payment/gateways/adyen/plugin.py (hunk 3 above)
Comment (korycins) [reply saleor/payment/gateways/adyen/plugin.py:100]: Added url

**Pull request merged**